tests/testthat/test-eg.R

```diff
@@ -2,7 +2,7 @@ context("testing setting endpoints to CCHP tiers")
 
 test_that("setting endpoints", {
   setBaseURL('prod')
-  expect_equal(pkg.env$item_json, "https://cida.usgs.gov/coastalchangehazardsportal/data/item/")
+  expect_equal(pkg.env$item_json, "https://marine.usgs.gov/coastalchangehazardsportal/data/item/")
```

Comment:
the cida.usgs.gov endpoint no longer works correctly due to internal config changes, so had to point it at actual prod url